src/common/lib/util/defaults.ts

```diff
@@ -43,7 +44,7 @@ type CompleteDefaults = IDefaults & {
   getHosts(options: NormalisedClientOptions): string[];
   checkHost(host: string): void;
   getRealtimeHost(options: ClientOptions, production: boolean, environment: string): string;
-  objectifyOptions(options: ClientOptions | string): ClientOptions;
+  objectifyOptions(options: ClientOptions | string, additionalPlugins?: ModularPlugins): ClientOptions;
```

Comment:
`additionalPlugins` param probably can be renamed to something like `includeModularPlugins`

Reply:
Will change to `modularPluginsToInclude`. Also I’m gonna switch the order of the spreads when creating `optionsObj` so that `modularPluginsToInclude` comes first — i.e. so that the user can override these defaults (it's pretty moot though, I don't see anyone doing this in practice).

Reply:
`modularPluginsToInclude` - this is just the name I was looking for, thank you!

> i.e. so that the user can override these defaults

Might be useful in testing in the future, so it's a welcome change. Also this is how I would expect library to perform if I ever passed my own plugins to it - they override default ones.